### DISCUSSION TITLE
Logs while running

### DISCUSSION THREAD

**Discussion by Oxfordjon** [simulot/immich-go#429] [Q&A] [answered]
How do I see what's happening once immich go is running?

I was watching it tick along nicely but closed the terminal window by mistake. From Unraid CPU usage, I can see it's running but I don't know how to see what's going on.

Thanks

**Comment by linuxturtle** [comment]
Mine, at the end of the run, pointed me to a log in ~/.cache/immich-go/.

**Comment by simulot** [comment, marked as answer]
look at the readme to file the location of the log files for your operating system.

https://github.com/simulot/immich-go#shared-options-with-all-commands
